tools/barista/src/builder/icons.ts

```diff
@@ -132,19 +128,19 @@ function getSvgWithoutFill(filePath: string): string {
 }
 
 export const iconsBuilder: BaPageBuilder = async () => {
-  if (!existsSync(ICONS_ROOT)) {
+  if (!existsSync(environment.iconsRoot)) {
     return [];
   }
 
   const changelogTemplate = getChangelogTemplate();
-  const iconFilePaths = getIconFilesByPath(ICONS_ROOT);
+  const iconFilePaths = getIconFilesByPath(environment.iconsRoot);
   const transformed: BaPageBuildResult[] = [];
   const iconOverviewData: BaIconOverviewItem[] = [];
 
   for (const filePath of iconFilePaths) {
     const iconName = basename(filePath, '.svg');
     const metadata: BaIconMetadata = JSON.parse(
-      readFileSync(join(ICONS_ROOT, `${iconName}.json`)).toString(),
+      readFileSync(join(environment.iconsRoot, `${iconName}.json`)).toString(),
```

Comment:
please check if exists before you read and parse (maybe you try the encoding thing than you get a string as return value instead of a buffer)